cmd/tracee-ebpf/main.go

```diff
@@ -57,17 +58,33 @@ func main() {
 			// for the rest of execution, use this debug mode value
 			debug := debug.Enabled()
 
+			// Logger Initialization
+			logger.Init(
+				&logger.LoggerConfig{
+					Writer: os.Stderr,
+					Level: func(debug bool) logger.Level {
+
+						if debug {
+							return logger.DebugLevel
+						}
+						return logger.InfoLevel
+
+					}(debug),
+					Encoder: logger.NewJSONEncoder(
+						logger.NewProductionConfig().EncoderConfig,
+					),
+				},
+			)
+
```

Comment:
This is just to show the Initialization capability. The logger (at package level) is already initialized to its [default](https://github.com/aquasecurity/tracee/pull/2110/files#diff-c1dc2eb1e149a1dc7ecd5cb474e167fb456eb7ad0ea0256cf62fba693a89af79R243).